Kconfig

```diff
@@ -50,6 +50,14 @@ config IMAGE_BOOT_PG2
 	  that it fits AND you have an external flash programmer at hand that
 	  allows to recover. Otherwise you risk to BRICK THE IOT2050!
 
+config IMAGE_BOOT_QEMU
+	bool "Firmware image for QEMU"
+	help
+	  Build the firmware image to use with QEMU.
+
+	  WARNING: Do not flash this image onto your device. Otherwise you
+	  risk to BRICK THE IOT2050!
+
```

Comment:
Nope, please not. Rather model that similar to isar-cip-core where U-Boot forms a helper for the QEMU start script. Maybe even consider a start script over lengthy README.

Reply:
Sure, I will write a start-qemu script. This is for generating firmware.bin file for qemu, so we will keep this image type right?

Reply:
https://gitlab.com/cip-project/cip-core/isar-cip-core/-/blob/master/kas/opt/ebg-swu.yml#L33 - this is how the u-boot firmware image is hooked up in isar-cip-core.

Reply:
If we switch to this model, then we have to remove iot2050-watchdog. Otherwise, there is a conflict for compiling u-boot for qemu.

Reply:
Done, switched that model and removed iot2050-watchdog.